libs/spark/src/Unstable_Button/Unstable_Button.stories.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { ChevronDown, AlertCircle } from '@prenda/spark-icons';
+import { sparkThemeProvider } from '../../stories';
+import { Box, Unstable_Button, Unstable_ButtonProps } from '..';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface Sb_Unstable_ButtonProps
+  extends Omit<
+    Unstable_ButtonProps,
+    | 'disableElevation'
+    | 'disableFocusRipple'
+    | 'centerRipple'
+    | 'disableRipple'
+    | 'disableTouchRipple'
+    | 'focusRipple'
+    | 'tabIndex'
+    | 'TouchRippleProps'
```

Comment:
You can omit these in the actual props. We don't want them to appear for consumers either.